src/publisher/views/publish/long-build.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import Layout from '../components/Layout';
+import DatasetStatus from '../../../shared/views/components/dataset/DatasetStatus';
+import T from '../../../shared/views/components/T';
+import { CubeBuildStatus } from '../../../shared/enums/cube-build-status';
+
+export default function LongBuild(props) {
+  const spinner = {
+    display: 'inline-block',
+    height: '50px',
+    width: '50px',
+    verticalAlign: 'middle',
+    marginLeft: '50px'
+  };
+  const showSpinner = () => {
+    switch (props.buildLogEntry.status) {
+      case CubeBuildStatus.Completed:
+      case CubeBuildStatus.Materializing:
+      case CubeBuildStatus.Failed:
+        return <></>;
+      default:
+        return <div className="loader" id="spinner" style={spinner}></div>;
+    }
+  };
+  const actionButton = () => {
+    switch (props.buildLogEntry.status) {
+      case CubeBuildStatus.Completed:
+        return (
+          <>
+            <a href={props.nextAction} className="govuk-button" id="action-button">
+              {props.t('buttons.continue')}
+            </a>
+          </>
```

Comment:
You only need react fragments `<>` if there's more than 1 top level element being returned, e.g. 

```tsx
  return (
    <div>something</div>
  );
```

doesn't required them, but 

```tsx
  return (
    <>
      <div>something</div>
      <div>something else</div>
    </>
  );
```
does.